### ISSUE TITLE
Hackney Adapter

### ISSUE THREAD

**Issue by swelham** [swelham/ivar#9]
Reminder to take a look at adding an adapter for using hackney directly from ivar.

**Comment by jjl** [comment]
We can reuse much of the code from the httpoison adapter, because it just passes a lot of stuff straight through to hackney as-is

**Comment by swelham** [comment]
Yes I was hoping that might be the case, I just haven't had much chance to look into it yet. I'm currently working on an HTTP/2 adapter using ace.

**Comment by jjl** [comment]
Yeah I saw your note. I've got some changes to make to the httpoison adapter for the multipart stuff we discussed when i get through with this current release (which is using httpoison for the bits we need multipart for at the minute).

**Comment by swelham** [comment]
Ok that's cool, thanks.
